src/components/common/LinkList.tsx

```diff
@@ -0,0 +1,16 @@
+import * as S from "./LinkList.styled";
+import Link from "./Link";
+import { LinkItem, LinkParam } from "../../model/Link";
+
+const SharedList = ({ listInfo, isSetting }: LinkParam) => {
+  return (
+    <S.ContentListBox>
+      {listInfo.map((link: LinkItem) => {
+        return <Link key={link.id} linkInfo={link} isSetting={isSetting} />;
+      })}
```

Comment:
```suggestion
      {listInfo.map((link: LinkItem) =><Link key={link.id} linkInfo={link} isSetting={isSetting} />
      )}
```
하나 리턴할 때는 왠만하면 바로 리턴하시는 걸 추천드려요!